frontend/src/services/graphQL.ts

```diff
@@ -1,15 +1,23 @@
 import axios, { AxiosError, AxiosResponse } from 'axios'
 import { getToken } from '../services/remote.it'
 import { store } from '../store'
-import { version } from '../../package.json'
 import { GRAPHQL_API, GRAPHQL_BETA_API } from '../shared/constants'
+import { version } from '../../package.json'
+
+export function getApiURL(isRest?: boolean): string {
```

Comment:
```suggestion
export function getApiURL(): string {
```

Adding restAPI to graphQL doesn't do anything. The restAPI is actually defined in `shared/constants.ts` as `API_URL` so this setting will have to override all uses of `API_URL` in the code base as well.